### PR TITLE
[ROS2] Fix deprecated PCL API: setFilterLimitsNegative -> setNegative

### PR DESCRIPTION
**Pull request by wxmerkt**
Cf.
https://github.com/PointCloudLibrary/pcl/commit/1036334ec2866aec5529a762f9c36f026562502f

### CODE REVIEW

**Comment by wxmerkt** [comment]
@wep21 Can you please check that these changes do not impact performance / functionality?

File: octomap_server/src/octomap_server.cpp [hunk 1]
```diff
@@ -450,14 +447,8 @@ void OctomapServer::insertCloudCallback(const PointCloud2::ConstSharedPtr cloud)
...
 
-
-    Eigen::Matrix4f sensor_to_base =
-      tf2::transformToEigen(sensor_to_base_transform_stamped.transform).matrix().cast<float>();
-    Eigen::Matrix4f base_to_world =
-      tf2::transformToEigen(base_to_world_transform_stamped.transform).matrix().cast<float>();
-
     // transform pointcloud from sensor frame to fixed robot frame
-    pcl::transformPointCloud(pc, pc, sensor_to_base);
+    pcl_ros::transformPointCloud(pc, pc, sensor_to_base_transform_stamped);
```
Comment (wep21) [inline octomap_server/src/octomap_server.cpp:451]: @wxmerkt Do you have any reason to migrate transform method from pcl to pcl_ros?

**Comment by wep21** [comment]
It seems to work fine.
image

File: octomap_server/src/octomap_server.cpp (hunk 1 above)
Comment (wxmerkt) [reply octomap_server/src/octomap_server.cpp:451]: It's no longer available in the new Rolling version

**Comment by wxmerkt** [comment]
Thank you @wep21

**Comment by wep21** [comment]
@wxmerkt Could you create a new release for ros2?

**Comment by wxmerkt** [comment]
@wep21: Done - thank you for the ping :-)

**Pull request merged**